cpp/log/ct_extensions_test.cc

```diff
@@ -166,16 +166,20 @@ TEST_F(CtExtensionsTest, TestPoisonExtension) {
 TEST_F(CtExtensionsTest, TestPrecertSigning) {
   // Sanity check
   Cert simple_ca_cert(simple_ca_cert_);
-  EXPECT_EQ(Cert::FALSE, simple_ca_cert.HasExtendedKeyUsage(
-                             cert_trans::NID_ctPrecertificateSigning));
+  StatusOr<bool> simple_ca_eku_status = simple_ca_cert.HasExtendedKeyUsage(
+      cert_trans::NID_ctPrecertificateSigning);
+  EXPECT_TRUE(simple_ca_eku_status.ok() &&
+              simple_ca_eku_status.ValueOrDie() == false);
```

Comment:
In tests, could probably *use* the implicit `OrDie`, and just go `EXPECT_FALSE(simple_ca_eku_status.ValueOrDie());`?